sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeActionsResult.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
-    /// The results of executing multiple actions in a set of documents.
+    /// The results of executing multiple actions on a set of documents.
     /// </summary>
     public class AnalyzeActionsResult
     {
```

Comment:
the properties docstrings in here could use the reference type too i.e.`<see cref="ExtractKeyPhrasesActionResult"/>`